joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/CallTests.scala

```diff
@@ -187,4 +219,38 @@ class CallTests extends PhpCode2CpgFixture {
     val call = cpg.call("test").head
     call.code shouldBe "test(...$args)"
   }
+
+  "chained calls should alias calls in receivers and bases" in {
```

Comment:
The hole tmp variable creation can be removed because in PHP the receiver and the this argument are always the same and you can just reference the same AST subtree.

Reply:
So, should we remove the `base` altogether? From what I understand, if the base and receiver are the same AST subtree the `tmp` stuff is there to reduce duplication between the two.

Reply:
Please add one tree for the this argument and point to one argument and one receiver edge to it. No extra base tree.